app/src/server.js

```diff
@@ -0,0 +1,80 @@
+import * as createServers from 'create-servers'
```

Comment:
Also add a little doc explaining this file. What is this server? Why we are spawning it?